client/components/dashboard/Borrowed.js

```diff
@@ -0,0 +1,63 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import { Button, Col } from 'react-materialize';
+
+
+const Borrowed = (props) => {
+  const books = props.borrowedBooks && props.borrowedBooks.length ?
+    props.borrowedBooks.map(book =>
+      <Col key={book.title} s={12} className="book-info grey-text">
+        <div className="card horizontal">
+          <div className="card-image">
+            <img
+              src={book.cover || 'https://segunolalive.github.io/helloBooks/templates/images/eloquentjs_cover.png'}
+              alt={book.title || 'N/A'}
+            />
+          </div>
+          <div className="card-stacked">
+            <div className="card-content">
+              <h6 className="bold-text black-text">
+                {book.title.toUpperCase() || null}
```

Comment:
Nitpick: why is there a value defaulting to null if the title property doesn't exist.